src/main.cc

```diff
@@ -86,6 +87,28 @@ class Spellchecker : public Nan::ObjectWrap {
     }
   }
 
+  static NAN_METHOD(CheckSpellingCallback) {
+    Nan::HandleScope scope;
+    if (info.Length() < 2) {
+      return Nan::ThrowError("Bad argument");
+    }
+
+    Handle<String> string = Handle<String>::Cast(info[0]);
+    if (!string->IsString()) {
+      return Nan::ThrowError("Bad argument");
+    }
+
+    Nan::Callback *callback = new Nan::Callback(info[1].As<v8::Function>());
```

Comment:
Oh, reading further down in `spellchecker.js`, it looks like the callback-based one is already not exported. In that case, I think it's fine to leave it as is.